@here/harp-mapview/lib/MapView.ts

```diff
@@ -2620,6 +2660,92 @@ export class MapView extends THREE.EventDispatcher {
         this.m_env.entries.$frameNumber = this.m_frameNumber;
     }
 
+    /**
+     * Update the directional light camera. Note, this requires the cameras to first be updated.
+     */
+    private updateLights() {
+        // TODO: HARP-9479Globe doesn't support shadows.
+        if (
+            !this.shadowsEnabled ||
+            this.directionalLight === undefined ||
```

Comment:
There could be multiple directional lights, so you should iterate over all lights and check their type.